packages/compass-aggregations/src/utils/stage.js

```diff
@@ -190,4 +196,29 @@ export function getStageInfo(namespace, stageOperator, stageValue) {
  */
 export const isLastStageOutputStage = (pipeline) => {
   return isOutputStage(getLastStageOperator(pipeline));
+};
+
+/**
+ * @param {string} env
+ * @param {import('mongodb').MongoServerError | null} serverError
+ */
+ export const isMissingAtlasStageSupport = (env, serverError) => {
+  return !!(
+    ![ADL, ATLAS].includes(env) &&
+    serverError &&
+    [
+      // Unrecognized pipeline stage name
+      40324,
+      // The full-text search stage is not enabled
+      31082,
+    ].includes(Number(serverError.code))
+  );
+};
+
+/**
+ * Return a list of atlas only operators
+ * @param {string[]} operators 
+ */
+export const getAtlasOnlyOperators = (operators) => {
```

Comment:
Based on how you are using this wouldn't it make more sense to make this a `find` instead of `filter`?